static/scss/components/whatsnew.scss

```diff
@@ -0,0 +1,177 @@
+// The `header .user-area:not(.arbitrary-class) is there to make this rule
+// more specific than some rules that apply to `.header-link` elements,
+// which we want to overwrite (in particular, the `display` property):
+header .user-area:not(.arbitrary-class) .c-whatsnew-trigger {
+    flex-shrink: 0;
+    display: none;
+    align-items: center;
+    gap: $spacing-sm;
+    background-color: transparent;
+    border-style: none;
+    // This is to make sure the label is vertically aligned with other menu items,
+    // which have a 2px bottom border when they're the active page:
+    border-bottom: 2px solid transparent;
+    border-radius: $border-radius-lg;
+    cursor: pointer;
+    padding: $spacing-xs $spacing-sm;
+
+    @media screen and #{$mq-md} {
+        display: flex;
+    }
+
+    &.is-open {
+        background-color: $color-light-gray-20;
+        color: $color-dark-gray-20;
+    }
+
+    .c-whatsnew-trigger-pill {
+        display: none;
+        border-radius: 100%;
+        padding: $spacing-sm;
+        font-size: 14px;
+        width: 20px;
+        height: 20px;
+        align-items: center;
+        justify-content: center;
+        background-color: $color-yellow-40;
+        font-weight: 700;
+        font-style: normal;
+        color: $color-white;
+
+        &.is-visible {
+            display: flex;
+        }
+    }
+}
+
+.c-whatsnew-popup {
+    $arrow-height: 12px;
+    display: none;
+    position: absolute;
+    transform: translateX(-50%) translateY(calc(50% + #{$spacing-md}));
+    color: $color-black;
+    background-color: $color-white;
+    box-shadow: $box-shadow-sm;
+    border-radius: $border-radius-sm;
+    width: $content-sm;
+    margin-top: $arrow-height;
+
+    &.is-visible {
+        display: flex;
+    }
+
+
+    &::before {
+        content: "";
+        height: $arrow-height;
+        width: $arrow-height;
+        position: absolute;
+        top: -1 * $arrow-height / 2;
+        right: 33%;
+        transform: rotate(45deg);
+        background-color: $color-white;
+    }
+
+    .c-whatsnew-list {
+        // Overwrite the bottom margin set on <ul>s by _common.scss:
+        margin: 0;
+        padding: 0 $spacing-md;
+
+        &.is-hidden {
+            display: none;
+        }
+
+        li {
+            display: none;
+            padding: $spacing-sm 0;
+            border-bottom: 1px solid $color-light-gray-30;
```

Comment:
You'll want to add an extra rule to remove this for the last item. 

I made the border color red to show its visibility: 

<img width="487" alt="image" src="https://user-images.githubusercontent.com/2692333/156069486-2b18e648-1370-48a7-b82e-9da417c09caf.png">